pmlc/dialect/stdx/tests/i1StorageToi8.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: pmlc-opt -stdx-i1-storage-to-i8 %s | FileCheck %s
+
+module {
+  func @eltwise_add(%arg0: memref<10x20xf32>, %arg1: memref<10x20xf32>) {
+    %cst = constant 0.000000e+00 : f32
+    %0 = alloc() : memref<10x20xi1>
+    %c0 = constant 0 : index
+    %c10 = constant 10 : index
+    %c1 = constant 1 : index
+    loop.for %arg2 = %c0 to %c10 step %c1 {
+      %c0_3 = constant 0 : index
```

Comment:
You can reuse `%c0`, no need to create a new constant with the same value. It looks like maybe this code came from the output of something else? If so, try grabbing this sample after the CSE and Canonicalize passes were used.

Reply:
Yes it was generated without CSE and Canonicalize, will update the code with usage of these passes.